src/pipe-controller.ts

```diff
@@ -40,14 +41,24 @@ export default class PipeController {
     const bottomPipe = new Pipe(this.ctx, this.width, this.canvas.height - bottomHeight, x, bottomHeight)
 
     this.pipes.push(...[topPipe, bottomPipe])
+    this.trackPassedPipes.push(topPipe)
   }
 
   collideWith = (sprite: Collideable) => {
     return this.pipes.some(pipe => pipe.collideWith(sprite))
   }
 
+  hasPassed = (sprite: Collideable) => {
```

Comment:
I made this type to track all collideable objects, but i feel like we should make sure this is in fact the Hummingbird sprite that is passing by, as what determines points is the Hummingbird passing the pipe, technically I think a standing still pipe would count points still right? It's nitpicky but maybe just for good coding we validate